scenarios/kubernetes_e2e.py

```diff
@@ -0,0 +1,204 @@
+
+    # Get golang into our PATH so we can run e2e.go
+    os.environ['PATH'] += ':/usr/local/go/bin'
+
+    # dockerized-e2e-runner goodies setup
+    workspace = lookup.setdefault('workspace', os.getcwd())
+    artifacts = '%s/_artifacts' % workspace
+    os.makedirs(artifacts)
+
+    e2e_image_tag = 'v20170104-9031f1d'
+    e2e_image_tag_override = '%s/hack/jenkins/.kubekins_e2e_image_tag' % workspace
+    if os.path.isfile(e2e_image_tag_override):
+        with open(e2e_image_tag_override, 'r') as tag:
+            e2e_image_tag = tag.read()
+
+    docker_env_ignore = [
+      'GOOGLE_APPLICATION_CREDENTIALS',
+      'GOROOT',
+      'HOME',
+      'PATH',
+      'PWD',
+      'WORKSPACE'
+    ]
+
+    # exec
+
+    print 'Starting %s...' % CONTAINER
+
+    cmd = ['docker', 'run', '--rm',
+      '--name=%s' % CONTAINER,
+      '-v', '%s/_artifacts":/workspace/_artifacts' % workspace,
+      '-v', '/etc/localtime:/etc/localtime:ro'
+    ]
+
+    # Rules for env var priority here in docker:
+    # -e FOO=a -e FOO=b -> FOO=b
+    # --env-file FOO=a --env-file FOO=b -> FOO=b
+    # -e FOO=a --env-file FOO=b -> FOO=a(!!!!)
+    # --env-file FOO=a -e FOO=b -> FOO=b
+    #
+    # So if you overwrite FOO=c for a local run it will take precedence.
+    #
+
+    if args.env_file:
+        for env in args.env_file:
+            cmd.extend(['--env-file', env])
+
+    gce_ssh = '/workspace/.ssh/google_compute_engine'
+    gce_pub = '%s.pub' % gce_ssh
+    service = '/service-account.json'
+
+    cmd.extend(['-v', '%s:%s:ro' % (args.gce_ssh, gce_ssh),
+                '-v', '%s:%s:ro' % (args.gce_pub, gce_pub),
+                '-v', '%s:%s:ro' % (args.service_account, service),
+                '-e', 'JENKINS_GCE_SSH_PRIVATE_KEY_FILE=%s' % gce_ssh,
+                '-e', 'JENKINS_GCE_SSH_PUBLIC_KEY_FILE=%s' % gce_pub,
+                '-e', 'GOOGLE_APPLICATION_CREDENTIALS=%s' % service])
+
+    # Boilerplate envs
+
+    # Skip gcloud update checking
+    cmd.extend(['-e', 'CLOUDSDK_COMPONENT_MANAGER_DISABLE_UPDATE_CHECK=\'true\''])
+    # Use default component update behavior
+    cmd.extend(['-e', 'CLOUDSDK_EXPERIMENTAL_FAST_COMPONENT_UPDATE=\'false\''])
+
+    cmd.extend(['-e', 'E2E_UP=%s' % args.e2e_up,
+                '-e', 'E2E_TEST=%s' % args.e2e_test,
+                '-e', 'E2E_DOWN=%s' % args.e2e_down,
+                '-e', 'E2E_NAME=%s' % args.e2e_name])
+
+    # AWS
+    cmd.extend(['-e', 'KUBE_AWS_INSTANCE_PREFIX=%s' % args.e2e_name])
+
+    # GCE
+    cmd.extend(['-e', 'INSTANCE_PREFIX=%s' % args.e2e_name,
+                '-e', 'KUBE_GCE_NETWORK=%s' % args.e2e_name,
+                '-e', 'KUBE_GCE_INSTANCE_PREFIX=%s' % args.e2e_name])
+
+    # GKE
+    cmd.extend(['-e', 'CLUSTER_NAME=%s' % args.e2e_name,
+                '-e', 'KUBE_GKE_NETWORK=%s' % args.e2e_name])
+
+    for key, value in os.environ.items():
+        if key not in docker_env_ignore:
+            cmd.extend(['-e', '%s=%s' % (key, value)])
+
+    cmd.extend([
```

Comment:
Is there a reason these two need to go at the end instead of with the other -e stanzas?